radio/src/pulses/pulses_avr.cpp

```diff
@@ -110,60 +110,61 @@ uint8_t *pulses2MHzWPtr = pulses2MHz;
 #define CTRL_REP_1CMD -3
 #define CTRL_REP_2CMD -6
 
-#define SETUP_PULSES_DURATION 1000/*500us*/
+#define SETUP_PULSES_DURATION 1000 /*500us*/
+uint8_t g_ppmPulsePolarity = 0; // Needed for Bit-bang PPM.
 
-// TIMER1_COMPA_vect used for PPM and DSM2=SERIAL
-uint8_t g_ppmPulsePolarity = 0;
-ISR(TIMER1_COMPA_vect) //2MHz pulse generation (BLOCKING ISR)
+// TIMER1_COMPA_vect used for PPM and DSM2=SERIAL.
+ISR(TIMER1_COMPA_vect) // 2MHz pulse generation (BLOCKING ISR).
 {
-  uint8_t dt = TCNT1L; // record Timer1 latency for DEBUG stats display
-  
-  // Call setupPulses only after REST pulse had been sent.
-  // Must do this before toggle PORTB to keep timing accurate
-  if (IS_DSM2_SERIAL_PROTOCOL(s_current_protocol[0]) || *((uint16_t*)pulses2MHzRPtr) == 0) {
-    if (!IS_DSM2_SERIAL_PROTOCOL(s_current_protocol[0])) {
-      OCR1A = SETUP_PULSES_DURATION;
-    }
-    setupPulses(); // does not sei() for setupPulsesPPM
-    heartbeat |= HEART_TIMER_PULSES;
-    return;
-  }
-
+	uint8_t dt = TCNT1L; // Record Timer1 latency for DEBUG stats display.
```

Comment:
I will comment on cosmetic errors only. We like to keep code clean and uniform. Use two spaces for indentation.